.ci/virtio-blk.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+
+function cleanup {
+    sleep 1
+    pkill -9 rv32emu
+}
+
+function ASSERT {
+    $*
+    local RES=$?
+    if [ $RES -ne 0 ]; then
+        echo 'Assert failed: "' $* '"'
+        exit $RES
+    fi
+}
+
+cleanup
+
+TIMEOUT=50
+OPTS=" -k build/linux-image/Image "
+OPTS+=" -i build/linux-image/rootfs.cpio "
+OPTS+=" -b build/minimal.dtb "
+OPTS+=" -x vblk:build/disk.img "
+RUN_LINUX="build/rv32emu ${OPTS}"
+
+dd if=/dev/zero of=build/disk.img bs=4M count=32
+mkfs.ext4 build/disk.img
```

Comment:
Check the presence of `mkfs.ext4`. That is, something like https://github.com/sysprog21/semu/blob/master/Makefile#L20-L32